[Workers] Gradual Deployments edits (#13768)
cloudflare/cloudflare-docs: content/workers/configuration/versions-and-deployments/_index.md

## Limits

### First upload

> You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions deploy --experimental-versions`](/workers/wrangler/commands/#upload) the first time you deploy a Worker will fail.

Author: ```suggestion
You must use [C3](/workers/get-started/guide/#1-create-a-new-worker-project) or [`wrangler deploy`](/workers/wrangler/commands/#deploy) the first time you create a new Workers project. Using [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload) the first time you upload a Worker will fail.
```

### Service worker syntax

Service worker syntax is not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload). You must use ES modules format. 

Refer to [Migrate from Service Workers to ES modules](/workers/reference/migrate-to-module-workers/#advantages-of-migrating) to learn how to migrate your Workers from the service worker format to the ES modules format.

### Smart Placement 

> Smart Placement is not support for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you have Smart Placement enabled on a script. 

Reviewer: ```suggestion
Smart Placement is not supporteded for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you have Smart Placement enabled on a Worker. 
```

Smart Placement support for versions will be coming in the near future.

### mTLS bindings 

> [mTLS bindings](/workers/runtime-apis/bindings/mtls/) are not support for versions that are uploaded through [`wrangler versions upload --experimental-versions`]. You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you mTLS bindings on a script. 

Reviewer: ```suggestion
[mTLS bindings](/workers/runtime-apis/bindings/mtls/) are not supported for versions that are uploaded through [`wrangler versions upload --experimental-versions`](/workers/wrangler/commands/#upload). You must use [`wrangler deploy`](/workers/wrangler/commands/#deploy) if you configured mTLS bindings in your Worker. 
```
